src/SignaturePad.WPF/InkPresenter.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections.Generic;
+
+namespace Xamarin.Controls
+{
+	partial class InkPresenter : System.Windows.Ink.StrokeCollection
```

Comment:
Maybe rather than a new type, is it possible to just add a using?
`using InkPresenter = System.Windows.Ink.StrokeCollection`

Reply:
removed as redundant